community-roles.md

```diff
@@ -4,32 +4,34 @@ and the responsibilities associated with each role, where applicable.
-The Community Leader defines the governing vision of the Solid project, and channels that vision into the projects and initiatives created to realise it. They are responsible for selecting core contributors, approving official projects and initiatives, prioritising official project roadmaps, and have the final say on project disagreements. The Community Leader has admin rights of Solid tools and can optionally attend the weekly recurring community support meeting.
+The Community Leader defines the governing vision of the Solid project. The Community Leader is responsible for approving the descriptions of the responsibilites of the Solid team roles as well as appointing individuals to the roles. The Community Leader is also responisble for approving changes to the Solid specifications and [official projects led by Solid](https://github.com/orgs/solid/projects). If there are differences of opinion, the Community Leader has the final say on the way forward. The Community Leader has admin rights of all Solid repositories. The Community Leader can optionally attend the weekly recurring community support meeting, project meetings, and user testing meeting.
 
 ## Community Manager
-The Community Leader defines the governing vision of the Solid project, and channels that vision into the projects and initiatives created to realise it. They are responsible for selecting core contributors, approving official projects and initiatives, prioritising official project roadmaps, and have the final say on project disagreements. The Community Leader has admin rights of Solid tools and can optionally attend the weekly recurring community support meeting.
+The Community Manager implements the governing vision of the Solid project. The Community Manager collects information from the Solid Team and Solid Community and puts it forward to the Community Leader as a suggestion for final approval. The Community Manager has admin rights of all Solid repositories and is responisble. The Community Manager is responisble for sharing the agenda prior to the community support meeting, incorporating suggestions to the agenda, sharing the minutes after the community support meeting and moderating the community support meeting. 
 
-## Repositories Manager
-The Repositories Manager controls Github, NPM, and other Solid organisation properties related to the storage, management, and distribution of official Solid projects. They are responsible for keeping these properties clean and well-organised on behalf of the Solid community. The Repositories Manager has admin rights of Solid tools must attend the weekly recurring community support meeting.
+## Repository Manager
+The Repository Manager is responisble for keeping the properties of their specific repository well-organised and communicating any changes clearly and publicly. The Respoitory Manager has admin rights of the repositosry they manage and is expected to attend the weekly recurring community support meeting.
```

Comment:
typo 'Respoitory' and 'repositosry'

Reply:
also `responisble` typo

Reply:
Include: "Repository Managers are responsible for moderating conversations to find consensus and merging pull requests on their respective repositories."